public/css/sample.css

```diff
@@ -0,0 +1,158 @@
+/* colors -> https://coolors.co/e54b4b-ebebd3-464655-e4b363-e8e9eb */
+
+@import url('https://fonts.googleapis.com/css?family=Quicksand:400,700');
```

Comment:
Why do you need to do this again? You are already referencing it in your **head.jade** file.

Is this file being used and how is it loaded unto your html file?

Reply:
I didn't work on this file at all. I only worked on home.css for the landing page. However, I will remove it